requirements/requirements.txt

```diff
@@ -4,6 +4,9 @@
 #
 #    'requirements/deps compile' (for details see requirements/README)
 #
+# TODO: remove this in the next PR
```

Comment:
Please remove this.

Reply:
If there is no diff in the requirements, the action will not trigger.
After the license checker is in-place, those changes would rightfully only be dependency-changes,
but for the initial PR other changes are necessary.
The TODO is thus meant for after this PR is merged.

I could also change the diff to a simple newline, that could theoretically stay in there indefinitely

Reply:
Isn't it enough when we can prove this has been run in this PR? If not, then this is fine with me.

Reply:
Now there is only a newline with a hashtag in the diff

Reply:
👍